src/styles/card.css

```diff
@@ -0,0 +1,83 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin-bottom: 40px;
+}
+
+.card__description {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-content: center;
+  align-items: center;
+  text-align: center;
```

Comment:
https://user-images.githubusercontent.com/102182808/207008350-4bfa20dd-45c8-4b81-bcde-7f2fbdfe6d0c.mov

remove all three centered properties and nothing will change